### ISSUE TITLE
Bug: JSON tags ignored

### ISSUE THREAD

**Issue by ArnoSen** [aristanetworks/goeapi#31]
I ran your example program in README.md in the root:
```
package main

import (
        "fmt"

        "github.com/aristanetworks/goeapi"
)

type MyShowVlan struct {
        SourceDetail string          `json:"sourceDetail"`
        Vlans        map[string]Vlan `json:"vlans"`
}

type Vlan struct {
        Status     string               `json:"status"`
        Name       string               `json:"name"`
        Interfaces map[string]Interface `json:"interfaces"`
        Dynamic    bool                 `json:"dynamic"`
}

type Interface struct {
        Annotation      string `json:"annotation"`
        PrivatePromoted bool   `json:"privatePromoted"`
}

func (s *MyShowVlan) GetCmd() string {
        return "show vlan configured-ports"
}

func main() {
        node, err := goeapi.ConnectTo("dut")
        if err != nil {
                panic(err)
        }

        sv := &MyShowVlan{}

        handle, _ := node.GetHandle("json")
        handle.AddCommand(sv)
        if err := handle.Call(); err != nil {
                panic(err)
        }

        for k, v := range sv.Vlans {
                fmt.Printf("Vlan:%s\n", k)
                fmt.Printf("  Name  : %s\n", v.Name)
                fmt.Printf("  Status: %s\n", v.Status)
        }
}
```

It worked and I get output like:
```
Vlan:1012
  Name  : VLAN1012
  Status: active
Vlan:1013
  Name  : hack
  Status: active
Vlan:1
  Name  : default
  Status: active
Vlan:1010
  Name  : VLAN1010
  Status: active
Vlan:1011
  Name  : VLAN1011 
  Status: active
```

Now when I change e.g. a variable name in the Vlan struct like below:
```
type Vlan struct {
        Status     string               `json:"status"`
        Name1       string               `json:"name"`
        Interfaces map[string]Interface `json:"interfaces"`
        Dynamic    bool                 `json:"dynamic"`
}
```
I would expect that things work in the same way because there is a json tag. However, the output I get is:

```
Vlan:1012
  Name  : 
  Status: active
Vlan:1013
  Name  : 
  Status: active
Vlan:1
  Name  : 
  Status: active
Vlan:1010
  Name  : 
  Status: active
Vlan:1011
  Name  : 
  Status: active
```

For some reason the tag is ignored and it looks like it uses the variable name itself for the JSON Unmarshal instead of the tag information.

**Comment by ArnoSen** [comment]
The issue occurs both with Go 1.11 as well as 1.13.

**Comment by ArnoSen** [comment]
I did some more digging and found out that the `mapstructure` lib does the unmarshalling. 
When I set the mapstructure tag on the struct var, then the Name1 struct variable is unmarshalled correctly. 

```
type Vlan struct {
        Status     string               `json:"status"`
        Name1      string               `mapstructure:"name"`
        Interfaces map[string]Interface `json:"interfaces"`
        Dynamic    bool                 `json:"dynamic"`
}
```

**Comment by cheynearista** [comment]
Addressed in PR #32 
Thanks @ArnoSen !
